style.css

```diff
@@ -151,3 +156,274 @@ li a {
 li a:hover {
   opacity: 0.7;
 }
+
+#works {
+  display: grid;
+  margin-top: 110px;
+}
+
+.cards {
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  width: 327px;
+  height: 552px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  box-sizing: border-box;
+  border-radius: 16px;
+  margin: 40px 12px;
+}
+
+#first-card-item {
+  margin-top: 150px;
+}
+
+#snapshot1 {
+  position: static;
+  width: 295px;
+  height: 220px;
+  left: 16px;
+  top: 16px;
+  background: url("./utils/first-card.png");
+  border-radius: 8px;
+  margin-top: 12px;
+  margin-bottom: 12px;
+  background-size: cover;
+}
```

Comment:
- [ ] Adjust your styles here, (especially the margins), so that the image can fully appear instead of showing half an image.

Reply:
I arranged margins and extended image sizes.

Reply:
:+1: This issue has been addressed successfully :heavy_check_mark: 

:warning: But all of the cards should have an image either background or just img tag, looking at [this live version]() of your portfolio, you will see that some projects are missing imgs